src/libraries/System.Private.CoreLib/src/System/Globalization/TimeSpanParse.cs

```diff
@@ -246,15 +246,12 @@ internal void BackOne()
                 if (_pos > 0) --_pos;
             }
 
-            internal char NextChar
+            internal char NextChar()
             {
-                get
-                {
-                    int pos = ++_pos;
-                    return (uint)pos < (uint)_value.Length ?
-                        _value[pos] :
-                        (char)0;
-                }
+                int pos = ++_pos;
```

Comment:
For some reason I thought we conventionally do post fix in such cases

Reply:
That would change the meaning. Can you clarify?

Reply:
Reading too fast 😀